roofit/roofitcore/src/RooNLLVar.cxx

```diff
@@ -225,9 +225,11 @@ RooNLLVar::RooNLLVar(const RooNLLVar& other, const char* name) :
 RooAbsTestStatistic* RooNLLVar::create(const char *name, const char *title, RooAbsReal& pdf, RooAbsData& adata,
             const RooArgSet& projDeps, const char* rangeName, const char* addCoefRangeName,
             Int_t nCPU, RooFit::MPSplit interleave, bool verbose, bool splitRange, bool binnedL) {
-  auto testStat = new RooNLLVar(name, title,
-      dynamic_cast<RooAbsPdf&>(pdf), adata,
-      projDeps, _extended, rangeName, addCoefRangeName, nCPU, interleave, verbose, splitRange, false, binnedL,
+  RooAbsPdf & thePdf = dynamic_cast<RooAbsPdf&>(pdf);
+  // check if pdf can be extended
+  bool extendedPdf = _extended & thePdf.canBeExtended();
```

Comment:
```suggestion
  bool extendedPdf = _extended && thePdf.canBeExtended();
```

Reply:
Since the variables are bools it will be the same, but yes, maybe it is better to make it clearer to use `&&`.
Thank you for the review !